### PR TITLE
Add required section for server to start normally in provision (secure) mode

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
Currently server required the following:
- "PYTHONPATH" set in environment variable
- "authorization" section

### CODE REVIEW

File: nvflare/lighter/project.yml [hunk 1]
```diff
@@ -106,6 +103,20 @@ builders:
...
+          - relaxed
+      roles:
+        super: super user of system
+      groups:
+        relaxed:
+          desc: org group with relaxed policies
+          rules:
+            allow_byoc: true
+            allow_custom_datalist: true
+      disabled: false
```
Comment (IsaacYangSLA) [inline nvflare/lighter/project.yml:119]: I recall we need to remove authorization.json.  Not sure if we will keep this for release.

File: nvflare/lighter/impl/master_template.yml [hunk 2]
```diff
@@ -366,6 +366,8 @@ sub_start_svr_sh: |
   DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
   echo "WORKSPACE set to $DIR/.."
   mkdir -p $DIR/../transfer 
+  export PYTHONPATH=/local/custom:$PYTHONPATH
+  echo "PYTHONPATH is $PYTHONPATH"
```
Comment (IsaacYangSLA) [inline nvflare/lighter/impl/master_template.yml:370]: Actually, this is to fix this line new_env["PYTHONPATH"] = new_env["PYTHONPATH"] + ":" + app_custom_folder.  I believe a better way to fix it is new_env["PYTHONPATH"] = new_env.get("PYTHONPATH", "") + ":" + app_custom_folder

Comment (YuanTingHsieh) [reply nvflare/lighter/impl/master_template.yml:370]: let me make that changes thanks

File: nvflare/lighter/project.yml (hunk 1 above)
Comment (YuanTingHsieh) [reply nvflare/lighter/project.yml:119]: let's table this and discuss

Comment (YuanTingHsieh) [reply nvflare/lighter/project.yml:119]: @yanchengnv @yhwen do you have any inputs on this? thanks

Comment (yhwen) [reply nvflare/lighter/project.yml:119]: @nvidianz is working on the re-engineering of admin authorization. Until that work is ready, we still need this to work for secure mode properly.

**Review by yhwen — APPROVED** [review]

**Pull request merged**